README.md

```diff
@@ -1 +1,87 @@
-# check-handler-plugin
\ No newline at end of file
+# {{ .Name }}
+
+## Table of Contents
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Asset registration](#asset-registration)
+  - [Resource definition](#resource-definition)
+- [Functionality](#functionality)
+- [Installation from source](#installation-from-source)
+- [Additional notes](#additional-notes)
+- [Contributing](#contributing)
+
+## Overview
+{{ .Name }} is a template repository which wraps the [Sensu Plugins Go Library][2].
+To use this project as a template, click the "Use this template" button from the main project page.
+
+When writing or updating a plugin's README from this template, review the Sensu Community
+[plugin README style guide][3] for content suggestions and guidance.
+
+## Configuration
+
+### Asset registration
+
+Assets are the best way to make use of this plugin. If you're not using an asset, please consider
+doing so! If you're using sensuctl 5.13 with Sensu Backend 5.13 or later, you can use the following
+command to add the asset:
+
+```
+sensuctl asset add {{ .GithubUser }}/{{ .GithubProject }}
+```
+
+If you're using an earlier version of sensuctl, you can find the asset on the [Bonsai Asset Index]([https://bonsai.sensu.io/assets/{{ .GithubUser }}/{{ .GithubProject }}](https://bonsai.sensu.io/assets/{{ .GithubUser }}/{{ .GithubProject }})).
+
+### Resource definition
+
+```yml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: {{ .Name }}
```

Comment:
Instead of `.Name` here and the next few lines, perhaps we should use `.GithubProject` as `.Name` may contain a more human-readable project name (e.g. Chef Deregistration Handler vs. chef-deregistration-handler).